From 23441ed6d74b4eba68333c802890cb396721a6d6 Mon Sep 17 00:00:00 2001
From: Volker Sobek <reklov@live.com>
Date: Tue, 19 Aug 2014 15:26:56 +0200
Subject: Fix rotate dialog

The rotate dialog didn't show up any more, because of an ancient
"has-separator" property.

https://bugzilla.gnome.org/show_bug.cgi?id=735055
---
 data/nautilus-image-rotate.ui | 1 -
 1 file changed, 1 deletion(-)

diff --git a/data/nautilus-image-rotate.ui b/data/nautilus-image-rotate.ui
index c40ba9a..7a50bba 100644
--- a/data/nautilus-image-rotate.ui
+++ b/data/nautilus-image-rotate.ui
@@ -30,7 +30,6 @@
     <property name="border_width">12</property>
     <property name="title" translatable="yes">Rotate Images</property>
     <property name="type_hint">GDK_WINDOW_TYPE_HINT_DIALOG</property>
-    <property name="has_separator">False</property>
     <child internal-child="vbox">
       <object class="GtkVBox" id="dialog-vbox1">
         <property name="visible">True</property>
-- 
2.11.0

